packages/plugins/rudder_plugin_android/android/build.gradle

```diff
@@ -38,6 +38,6 @@ android {
 }
 
 dependencies {
-    implementation 'com.rudderstack.android.sdk:core:[1.8.1, 1.9.0)'
+    implementation 'com.rudderstack.android.sdk:core:[1.14.0, 1.15.0)'
```

Comment:
is this latest after recent relase?

Reply:
it is not the latest after release, will update it

Reply:
updated this